src/Layout/Presentation/ElectionOverview/ElectionOverview.tsx

```diff
@@ -39,8 +39,9 @@ export class ElectionOverview extends React.Component<ElectionOverviewProps, {}>
         const data: ElectionOverviewDatum[] = [];
         const currents = this.props.partyResults;
         const comparisons = this.props.comparisonPartyResults;
+        const shouldCalculateDifference = currents.length === comparisons.length;
         for (let i = 0; i < currents.length; i++) {
-            const difference = currents[i].totalSeats - comparisons[i].totalSeats;
+            const difference = shouldCalculateDifference ? currents[i].totalSeats - comparisons[i].totalSeats : 0;
```

Comment:
It would be good to move "currents[i].totalSeats - comparisons[i].totalSeats" into some kind of explaining variable.